### PR TITLE
feat: Add required constraint on InputText

### PR DESCRIPTION
**Pull request by Merkur39**
Added `*` at first on type attribute, defined input as required

### CODE REVIEW

File: src/utils/input.js [hunk 1]
```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
...
   return result
 }
+
+/**
+ * @param {number} valueLength - Length of input value
+ * @param {boolean} isRequired - If value is required
+ * @param {number} inputMaxLength - Minimum limit of length of the input value
+ * @returns {boolean}
+ */
+export const checkInputConstraints = (
```
Comment (Ldoppea) [inline src/utils/input.js:50]: It seems a bit weird to pass the length and not the raw value here. The method is name `checkInput` but it is more `checkLength`.

File: src/utils/input.js [hunk 2]
```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
 
   return result
 }
+
+/**
+ * @param {number} valueLength - Length of input value
+ * @param {boolean} isRequired - If value is required
+ * @param {number} inputMaxLength - Minimum limit of length of the input value
```
Comment (Ldoppea) [inline src/utils/input.js:47]: the parameter is named `max` but the comment is about `min`. Which one is correct?

File: src/utils/input.js [hunk 3]
```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
...
+export const checkInputConstraints = (
+  valueLength,
+  isRequired,
+  inputMaxLength
+) => {
+  let isValid = false
+
+  if (inputMaxLength) {
+    isValid =
+      valueLength === inputMaxLength || (valueLength === 0 && !isRequired)
```
Comment (Ldoppea) [inline src/utils/input.js:59]: You seems to compare strick equality with the length. Am I missing something?

As I read the code, the parameter should be named `expectedLength` or similar

File: src/utils/input.js (hunk 2 above)
Comment (Merkur39) [reply src/utils/input.js:47]: I'll rename it `expectedLength` as proposed below, because it's exactly that 👍

File: src/utils/input.js (hunk 1 above)
Comment (Merkur39) [reply src/utils/input.js:50]: It doesn't only test the length, I can propose something more verbose like `checkConstraintsOfIinput`.

**Review by Ldoppea — APPROVED** [review]

**Pull request merged**